sigs.yaml

```diff
@@ -1138,9 +1138,9 @@ sigs:
         company: Ticketmaster
     meetings:
     - description: Regular SIG Meeting
-      day: Thursday
-      time: "00:00"
-      tz: "UTC"
+      day: Wednesday
+      time: "16:00"
+      tz: "PT"
```

Comment:
Unfortunately, you have to actually use the format `"PT (Pacific Time)"` so that the time zone converter works